lib/mogrify.ex

```diff
@@ -166,6 +166,7 @@ defmodule Mogrify do
 
   defp normalize_arguments({"histogram:" <> option, nil}), do: ["histogram:#{option}"]
   defp normalize_arguments({"pango", params}), do: ["pango:#{params}"]
+  defp normalize_arguments({"plasma", params}), do: ["plasma:#{params}"]
```

Comment:
Does it makes sense to add a couple of tests for this following what has been done for pango? https://github.com/route/mogrify/blob/master/test/mogrify_test.exs#L164

cc @talklittle  :wink:

Reply:
@kpanic i've added a couple tests